prototype/src/main/webapp/view-meeting.html

```diff
@@ -4,6 +4,9 @@
     <meta charset="UTF-8" />
     <title>Meeting</title>
     <script src="viewMeeting.js"></script>
+    <script src="chooseLocation.js"></script>
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyAcLlZxKd7qtaKgdkko0zMJyKzV4jImaG8"></script>
```

Comment:
Avoid committing API keys - instead put it in a separate json file that does not get committed. Then, use JS to load that file, and use something similar to [this](https://developers.google.com/maps/documentation/javascript/overview#Dynamic) to dynamically insert the script.

Reply:
I've sent a seperate PR.